docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2378 @@
+{
```

Comment:
<p>"to obtain" instead of "to obtained"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='115'/>

Reply:
Thx! Solved in https://github.com/pymc-devs/pymc/pull/5721/commits/4ce059a6fe34c9ca2aade794a8af36efbb875258